apps/wing/src/commands/compile.ts

```diff
@@ -200,3 +201,18 @@ export async function compile(entrypoint: string, options: ICompileOptions) {
     }
   }
 }
+
+/**
+ * Resolves a list of plugin paths as absolute paths, using the current working directory
+ * if absolute path is not provided.
+ * 
+ * @param plugins list of plugin paths (absolute or relative)
+ * @returns list of absolute plugin paths or relative to cwd
+ */
+function resolvePluginPaths(plugins: string[]): string[] {
+  const resolvedPluginPaths: string[] = [];
+  for (const plugin of plugins) {
+    resolvedPluginPaths.push(isAbsolute(plugin) ? plugin : resolve(process.cwd(), plugin));
```

Comment:
I suspect `resolve(process.cwd(), absPath)` will just return the absolute path:

```suggestion
    resolvedPluginPaths.push(resolve(process.cwd(), plugin));
```

Reply:
Ah reading the docs for that now I think you are right, thats really slick!